packages/console/src/pages/EnterpriseSso/Guide/XmlFileUploader/index.module.scss

```diff
@@ -0,0 +1,56 @@
+@use '@/scss/underscore' as _;
+
+.fileInput {
+  display: none;
+}
+
+.preview {
+  display: flex;
+  max-width: 400px;
+  align-items: center;
+  flex-direction: row;
+  padding: _.unit(5) _.unit(5) _.unit(5) _.unit(4);
```

Comment:
nit: just curious why the padding left is different?

Reply:
by design lol